frequency-chain/provider-dashboard
# Theme update

## Goal

The goal of this PR is to pull in the Style Guide and use it where possible, and otherwise make the style match what is on Frequency.xyz and other branded sites.

Known bug: after changing networks the path is appended and not cleared first.  One result is the Nav indicator for the current page disappears. See #197 

Not included:  updates to styles, given we do not have mocks for the new style for this site.

Closes #195  

## Checklist


## How To Test the Changes
```shell
git fetch && git checkout theme-update
npm run dev
```
1. Click through steps to:  connect to network, connect a provider, start to Add an Account Id or Stake, change networks, Become a Provider
2. Make sure the style is consistent and all links are visible, modals can be canceled, etc.
